src/file/format/dhall.rs

```diff
@@ -0,0 +1,49 @@
+use std::collections::HashMap;
+use std::error::Error;
+
+use crate::value::{Value, ValueKind};
+
+pub fn parse(
+    uri: Option<&String>,
+    text: &str,
+) -> Result<HashMap<String, Value>, Box<dyn Error + Send + Sync>> {
+    // Parse a Dhall object value from the text
+    // TODO: Have a proper error fire if the root of a file is ever not a Table
```

Comment:
I guess we can resolve this right away, can't we?

Reply:
I shamelessly stole this from the JSON impl, including the comment. I figured having consistent error handling was important.

What is the best type of error to throw in this case?

Reply:
Actually, I noticed that the error is properly handled in the JSON5 implementation. I'll use that as a model